core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -138,6 +146,19 @@ public String description() {
     return "Prints rfile info";
   }
 
+  @SuppressWarnings("unchecked")
+  protected Class<? extends BiFunction<Key,Value,String>> getFormatter(String formatterClazz) {
+    try {
+      if (formatterClazz != null) {
+        return (Class<? extends BiFunction<Key,Value,String>>) this.getClass().getClassLoader()
+            .loadClass(formatterClazz).asSubclass(BiFunction.class);
+      }
+    } catch (ClassNotFoundException e) {
+      System.err.println("Could not find formatter class: " + formatterClazz);
```

Comment:
If the user's command-line cannot be satisfied (probably a typo in their class name), we should probably hard-fail by letting this exception fall through, rather than returning null and by effetively wasting time dumping a file in a format the user doesn't want.